server/handlers/handlers.go

```diff
@@ -60,3 +60,37 @@ func PlayerScores(game *stycoon.Game, w http.ResponseWriter, req *http.Request)
 	}
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 }
+
+func Commands(game *stycoon.Game, w http.ResponseWriter, req *http.Request) {
+	// read
+	body, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		log.Warn().Err(err).Msg("Error reading body")
+		http.Error(w, "can't read request body", http.StatusBadRequest)
+		return
+	}
+	// parse
+	commands, err := stycoon.ParseCommands(string(body))
+	if err != nil {
+		log.Warn().Err(err).Msg("Command parsing failed")
+		http.Error(w, "parsing failed", http.StatusBadRequest)
+		return
+	}
+	// process
+	errs := game.ProcessCommands(commands)
+	for _, _ = range errs {
+		b, err := json.Marshal(errs)
+		if err != nil {
+			log.Warn().Err(err).Msg("Json marshall failed")
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+		_, err = w.Write(b)
+		if err != nil {
+			log.Warn().Err(err).Msg("response write failed")
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
```

Comment:
i am setting this in common headers alongside CORS. so it is not necessary here. but we can fix that when we are merging the visualization.

Reply:
yes let's do it after we merge it